src/content/app-platform/defaulting-validation/index.md

```diff
@@ -109,6 +109,11 @@ Currently we validate:
 - All referenced configmaps and secrets exist.
 - The catalog has a matching AppCatalog CR.
 
+If app-operator finds a matching AppCatalogEntry CR named `{appCatalog}-{app}-{version}`, it will run more two validation.
+
+- Cloud provider compatibility (e.g. you can’t install kiam app in azure, for example)
```

Comment:
```suggestion
- Cloud provider compatibility (e.g. you can’t install the azure-ad-pod-identity app in AWS).
```

kiam is pretty well know but I just switched the example because it has Azure in the name. WDYT?

Reply:
Good catch, let's use azure app instead.